grc/gui/Application.py

```diff
@@ -311,7 +311,7 @@ def flow_graph_update(fg=flow_graph):
             flow_graph.move_selected(coords)
 
             # Set flow graph to heir block type
-            top_block = flow_graph.get_block("top_block")
+            top_block = flow_graph.get_block("default")
```

Comment:
Thank you for tracking this one down.  The only thing I might suggest here is replacing this hardcoded name with Constants.DEFAULT_FLOW_GRAPH_ID

Reply:
Thank you, applied the fix to 3.8 and now it’s possible to highlight blocks and right click to create a heir block.

Reply:
Adapted it accordingly